# 雷达相对整车坐标系原点的挂载位置信息
float32 mount_xpos
float32 mount_ypos
float32 mount_zpos

#雷达相对雷达安装位置整车坐标系方向的安装角度
float32 raw_yaw_angle
float32 raw_pitch_angle

#定义车辆的长宽高轴距
float32 car_width
float32 car_length
float32 car_height
float32 car_wheelbase

